Application/EdFi.Ods.AdminApi/Features/Applications/ReadApplication.cs

```diff
@@ -19,6 +19,11 @@ public void MapEndpoints(IEndpointRouteBuilder endpoints)
             .WithRouteOptions(b => b.WithResponse<ApplicationModel[]>(200))
             .BuildForVersions(AdminApiVersions.V1);
 
+        AdminApiEndpointBuilder.MapGet(endpoints, "/applications/{offset}/{limit}", GetApplicationsPaging)
+            .WithDefaultDescription()
+            .WithRouteOptions(b => b.WithResponse<ApplicationModel[]>(200))
+            .BuildForVersions(AdminApiVersions.V1);
+
         AdminApiEndpointBuilder.MapGet(endpoints, "/applications/{id}", GetApplication)
```

Comment:
@lchavarriaga 
We don't want the separate endpoint for including the offset and limit. This can be included in the GetApplications endpoint itselt. Please implement the same changes for other endpoints as well.